### ISSUE TITLE
GetFriendlyUrl overload with language will only work with ignoreContextMode true

### ISSUE THREAD

**Issue by jakke-korpelainen** [Geta/EPi.Extensions#29]
In an overload of GetFriendlyUrl() that includes a language parameter, the language will only be used if ignoreContextMode is set to true. Is this a bug or intended/crucial for the language to work?

```
public static string GetFriendlyUrl(this ContentReference contentReference, string language, bool includeHost = false, bool ignoreContextMode = false, UrlResolver urlResolver = null)
{
      if (ContentReferenceExtensions.IsNullOrEmpty(contentReference))
        return string.Empty;
      if (urlResolver == null)
        urlResolver = ServiceLocator.Current.GetInstance<UrlResolver>();
      string url = urlResolver.GetUrl(contentReference);
      if (ignoreContextMode)
        url = urlResolver.GetUrl(contentReference, language, new VirtualPathArguments()
        {
          ContextMode = ContextMode.Default
        });
      if (includeHost)
        return StringExtensions.GetExternalUrl(url);
      return url;
}
```

**Comment by valdisiljuconoks** [comment]
Sounds like a bug. PR?

**Comment by frederikvig** [comment]
Thanks fixed in version 1.5.2
